src/Catalyst.Core.Modules.Dfs.Tests/UnitTests/RepositoryOptionsTest.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using Xunit;
+using Assert = Microsoft.VisualStudio.TestTools.UnitTesting.Assert;
+
+namespace Catalyst.Core.Modules.Dfs.Tests.UnitTests
+{
+    public class RepositoryOptionsTest
+    {
+        [Fact]
+        public void Defaults()
+        {
+            var options = new RepositoryOptions();
+            Assert.IsNotNull(options.Folder);
+        }
+
+        [Fact]
+        public void Environment_Home()
+        {
+            var names = new string[] {"IPFS_PATH", "HOME", "HOMEPATH"};
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the array type; it is redundant.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4612924)